src/lib/components/UrlChooser/index.js

```diff
@@ -0,0 +1,185 @@
+import {html} from "lit-element";
+import {store} from "../../store";
+import {BaseElement} from "../BaseElement";
+import {userRef, firebase} from "../../fb";
+
+/* eslint-disable require-jsdoc */
+class UrlChooser extends BaseElement {
+  static get properties() {
+    return {
+      href: {type: String},
+      switching: {type: Boolean, reflect: true},
+      active: {type: Boolean},
+    };
+  }
+
+  constructor() {
+    super();
+    this.href = null; // when signed out or waiting for Firestore, this is null
+    this.switching = true;
+    this.active = false;
+
+    // non-properties
```

Comment:
Are these private properties? If so maybe we underscore them?

Reply:
Done